src/ucp/core/ucp_worker.c

```diff
@@ -3159,21 +3202,16 @@ void ucp_worker_keepalive_add_ep(ucp_ep_h ep)
 {
     ucp_worker_h worker = ep->worker;
 
-    ucs_assert(ep->cfg_index != UCP_WORKER_CFG_INDEX_NULL);
-
-    if ((ep->flags & UCP_EP_FLAG_INTERNAL) ||
-        (ucp_ep_config(ep)->key.ep_check_map == 0) ||
-        !ucp_worker_keepalive_is_enabled(worker)) {
-        ucs_trace("ep %p flags 0x%x cfg_index %d: not using keepalive, "
-                  "err_mode %d ep_check_map 0x%x",
-                  ep, ep->flags, ep->cfg_index, ucp_ep_config(ep)->key.err_mode,
-                  ucp_ep_config(ep)->key.ep_check_map);
+    if (ucp_ep_config(ep)->key.keepalive_lane == UCP_NULL_LANE) {
+        ucs_trace("ep %p flags 0x%x cfg_index %d: not added to keepalive,"
+                  " err_mode %d", ep, ep->flags, ep->cfg_index,
```

Comment:
plz rephrase. Smthg, like `keepalive lane is not set`

Reply:
done